docs/user_guide/service_config.md

```diff
@@ -977,6 +977,37 @@ Example:
 admissionController.accessControl.externalGroups: "^sales$,^marketing$,^admin-"
 ```
 
+### Compresion setting
+
+The data in ConfigMap cannot exceed 1 MiB. YuniKorn supports Gzip algorithm to decompress data in the `binaryData` field.
+If a key ends with `.gz`, YuniKorn decodes the value by base64 and decompress it by Gzip algorithm.
+If both `queues.yaml` setting is set in `data` field and `queues.yaml.gz` setting is set in `binaryData` field, the value in `binaryData` field will be used.
```

Comment:
Instead: If a value is set in both the `data` and `binaryData` sections, the value in the `binaryData` section will be used.

Reply:
We don't want to log a warning because it's not uncommon that a default queues.yaml is provided via Helm in the yunikorn-defaults Condit so that is then overridden by a binaryData version in yunikorn-configs. It's not possible to remove a config value so having both would create an "unfixable" warning.

Reply:
> it's not uncommon that a default queues.yaml is provided via Helm in the yunikorn-defaults configmap that is then overridden by a binaryData version in yunikorn-configs. 

make sense to me. thanks